### ISSUE TITLE
Update funding oss doc?

### ISSUE THREAD

**Issue by rdela** [buttondown-email/docs#24]
https://docs.buttondown.email/behind-the-scenes/funding
https://github.com/buttondown-email/docs/blob/main/pages/behind-the-scenes/funding.mdx

Is there newer data to incorporate?

looks like [latest commit](https://github.com/buttondown-email/docs/commit/6ee61aaf8e5707265e463d738db099784f5f5eb8) was Jul 26, 2021

**Comment by jmduke** [comment]
Yes, thank you for the bump! I have a half-baked PR to automate this that fell by the wayside. Leaving this open until I pull the numbers for the most recent month.
